src/devices/somfy.ts

```diff
@@ -8,6 +8,13 @@ import type {DefinitionWithExtend} from "../lib/types";
 const e = exposes.presets;
 
 export const definitions: DefinitionWithExtend[] = [
+    {
+        zigbeeModel: ["Tilt Only 50 WF TiltOnly"],
+        model: "Tilt Only 50 WF TiltOnly",
```

Comment:
Is there a better model number written on the device or product page? If yes please use that, also update the picture name in the docs with this.